### ISSUE TITLE
Sorting options

### ISSUE THREAD

**Issue by NikunjKhangwal** [butzist/ActivityLauncher#254]
Add a sort icon beside search icon to sort apps with name (default), installed date & update time

**Comment by butzist** [comment]
Good idea, but unless we get a PR, not likely before PlayStore release

**Comment by NikunjKhangwal** [comment]
Fair enough

**Comment by butzist** [comment]
Do you have any ideas for UI integration?

**Comment by NikunjKhangwal** [comment]
Drop down menu
I think text is enough. There are material icons posted by Google but it only has icon for sorting by name.
